test/e2e/network/service.go

```diff
@@ -83,6 +82,11 @@ const (
 	// AffinityConfirmCount is the number of needed continuous requests to confirm that
 	// affinity is enabled.
 	AffinityConfirmCount = 15
+
+	kubeProxyLabelName  = "kube-proxy"
+	kubeAPIServerLabelName = "kube-apiserver"
+	clusterComponentKey = "component"
+	clusterAddonLabelKey  = "k8s-app"
```

Comment:
are these labels standard?
can you add a comment explaining the origin?

by the way, the go-fmt on this file, it seems need to be indented

Reply:
the const str "kube-apiserver"  and "component" is appear before. And I just set them as const var in this PR. For go-fmt I will fix it right now. And also the comment explaining.